hal/src/thumbv7em/nvm.rs

```diff
@@ -326,6 +326,7 @@ impl Nvm {
     ///
     /// If `destination_address` is not word-aligned, an error is returned.
     #[inline]
+    #[allow(clippy::missing_safety_doc)]
```

Comment:
I'd almost prefer not fixing these. It would be ideal if we could fill out `Safety` documentation.. I understand you can't fill all of them out as you didn't write this code

Reply:
Agreed.

Reply:
Awesome, thank you very much!